### DISCUSSION TITLE
Help with query

### DISCUSSION THREAD

**Discussion by 4l3j4ndr0** [adonisjs/core#2537] [Help]
Hello, i need your help.
I have the next query 
![image](https://user-images.githubusercontent.com/18580284/116824458-fec4ab80-ab4f-11eb-846a-c9448ea3294f.png)

And i have the next error

![image](https://user-images.githubusercontent.com/18580284/116824516-48ad9180-ab50-11eb-8454-e32a4b3a867e.png)

When I remove the condition, it actually works and returns the distance value.

If someone can give me a suggestion on how to do it, I thank you.

**Comment by Obapelumi** [comment]
Could you share your database schema? Looks like one of the tables you’re querying does not have the distance column

**Reply by alejo9424** [reply]
Hola @Obapelumi la columna no existe como vez es un alias que le doy a esa operación.
